controller/iteration_test.go

```diff
@@ -422,8 +462,16 @@ func createSpaceAndRootAreaAndIterations(t *testing.T, db application.DB) (space
 	var rootIterationObj iteration.Iteration
 	var otherIterationObj iteration.Iteration
 	application.Transactional(db, func(app application.Application) error {
+		owner := &account.Identity{
+			ID:           uuid.NewV4(),
+			Username:     "new-space-owner-identity",
+			ProviderType: account.KeycloakIDP,
+		}
+		errCreateOwner := app.Identities().Create(context.Background(), owner)
+		require.Nil(t, errCreateOwner)
 		spaceObj = space.Space{
-			Name: "Test 1-" + uuid.NewV4().String(),
+			Name:    "Test 1-" + uuid.NewV4().String(),
```

Comment:
Yes now using in https://github.com/almighty/almighty-core/pull/1347/commits/dd3159268e68f2e0036df69c31f208e104325781